test/lib/auth-client.js

```diff
@@ -6,54 +6,95 @@ const nock = require('nock');
 
 const AuthClient = require('../../lib/auth-client');
 
+const CLIENT_ID = '4cf82729-4275-46d9-9255-8437ba777151';
+const INVALID_CLIENT_ID = '4cf82729-4275-46d9-9255-87ba151';
+const CLIENT_SECRET = '4cf82729-4275-46d9-9255-8437ba777151';
+
 test('constructor', function(t) {
 
   const client = new AuthClient({
-    clientId: 'CLIENT_ID',
-    clientSecret: 'CLIENT_SECRET',
+    clientId: CLIENT_ID,
+    clientSecret: CLIENT_SECRET,
     redirectUri: 'https://insurance.co/callback',
     scope: ['read_odometer', 'read_vehicle_info'],
   });
 
-  t.is(client.clientId, 'CLIENT_ID');
-  t.is(client.clientSecret, 'CLIENT_SECRET');
+  t.is(client.clientId, CLIENT_ID);
+  t.is(client.clientSecret, CLIENT_SECRET);
   t.is(client.redirectUri, 'https://insurance.co/callback');
   t.deepEqual(client.scope, ['read_odometer', 'read_vehicle_info']);
   t.is(client.development, false);
   t.true('request' in client);
 
 });
 
-test('constructor - development', function(t) {
+test('constructor - validation error', function(t) {
 
-  const client = new AuthClient({
-    clientId: 'CLIENT_ID',
-    clientSecret: 'CLIENT_SECRET',
+  t.throws(() => new AuthClient({
+    clientId: 'f3266b17-961d-4295-8544-054c7bd94fbb',
     redirectUri: 'https://insurance.co/callback',
     scope: ['read_odometer', 'read_vehicle_info'],
-    development: true,
-  });
+  }), Error);
```

Comment:
nit: no need to add the 2nd arg unless you want to check something more specific, it implicitly already is an error